indexer/api/api.go

```diff
@@ -45,6 +36,23 @@ const (
 	WithdrawalsPath = "/api/v0/withdrawals/"
 )
 
+// Api ... Indexer API struct
+// TODO : Structured error responses
+type APIService struct {
+	log    log.Logger
+	router *chi.Mux
+
+	bv      database.BridgeTransfersView
+	dbClose func() error
+
+	metricsRegistry *prometheus.Registry
+
+	apiServer     *httputil.HTTPServer
+	metricsServer *httputil.HTTPServer
+
+	stopped atomic.Bool
```

Comment:
The docs for atomic package sound like we aren't supposed to be using it:

> Package atomic provides low-level atomic memory primitives useful for implementing synchronization algorithms.
> 
> These functions require great care to be used correctly. Except for special, low-level applications, synchronization is better done with channels or the facilities of the [sync](https://pkg.go.dev/sync) package. Share memory by communicating; don't communicate by sharing memory.

Reply:
It's a generic warning, and they're right about communication, but we don't want data-communication with channels here, it's just a basic variable I want to set to track the state